README.md

```diff
@@ -161,9 +161,9 @@ JTA and JTA in turn a superset of Core.
 
 3rd party dependency management is done via maven. Note that versions of most 3rd party components are resolved via the JBossAS component-matrix
 pom.xml, even when building standalone releases. The version of JBossAS to use is determined by the top level pom.xml
-You may need to set up maven to use the jboss.org repositories: http://community.jboss.org/wiki/MavenGettingStarted-Users
+You may need to set up maven to use the jboss.org repositories: https://community.jboss.org/wiki/MavenGettingStarted-Users
 
 Maven is provided in the tools/maven section, though later versions of this may work. Download locations are:
```

Comment:
Ondra has just [submitted a PR](https://github.com/jbosstm/narayana/pull/1868), which is ready to be merged, that removes maven from our tools section and just needs merging. So I think this paragraph can be deleted.

Reply:
+1, I've just missed this fact :-)

Reply:
Thanks @mmusgrov done the changes :) check once